### PR TITLE
Use a configuration file to do Email Notification Server configuration

### PR DESCRIPTION
**Pull request by sophiesongge**

### CODE REVIEW

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SendMail.java [hunk 1]
```diff
@@ -1,40 +1,78 @@
...
 
-    static {
-        Properties properties = System.getProperties();
-        if (properties.getProperty(SMTP_HOST_PROPERTY_NAME) == null) {
-            properties.setProperty(SMTP_HOST_PROPERTY_NAME, DEFAULT_SMTP_HOST);
-        }
-        session = Session.getDefaultInstance(properties);
-    }
+        Session session = Session.getInstance(properties, new javax.mail.Authenticator() {
+            protected PasswordAuthentication getPasswordAuthentication() {
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/SendMail.java]: MAJOR Add the "@Override" annotation above this method signature [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)

File: scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java [hunk 2]
```diff
@@ -1,5 +1,6 @@
 package org.ow2.proactive.scheduler.core;
 
+import java.io.IOException;
```
Comment (sonarqube-activeeon) [inline scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java]: MINOR Remove this unused import 'java.io.IOException'. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 6 issues
* MAJOR 1 major
* MINOR 5 minor

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MINOR [JobEmailNotification.java#L94](https://github.com/ow2-proactive/scheduling/blob/ec8447e69b54856469e5faa3e1c2dde8bcf0ab0e/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java#L94): Remove the declaration of thrown exception 'org.ow2.proactive.scheduler.core.JobEmailNotificationException' which is a runtime exception. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)
1. MINOR [JobEmailNotification.java#L102](https://github.com/ow2-proactive/scheduling/blob/ec8447e69b54856469e5faa3e1c2dde8bcf0ab0e/scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java#L102): Remove the declaration of thrown exception 'org.ow2.proactive.scheduler.core.JobEmailNotificationException' which is a runtime exception. [rule](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)

**Comment by sophiesongge** [comment]
retest this please

**Pull request merged**